pkg/volume/csi/nodeinfomanager/nodeinfomanager_test.go

```diff
@@ -637,6 +637,110 @@ func TestInstallCSIDriverExistingAnnotation(t *testing.T) {
 	}
 }
 
+func TestValidateCSINodeInfo(t *testing.T) {
+	testcases := []struct {
+		name      string
+		nodeInfo  *csiv1alpha1.CSINodeInfo
+		expectErr bool
+	}{
+		{
+			name: "multiple drivers with same ids and different topology keys",
```

Comment:
There's no requirement around NodeIDs right? Currently the NodeIDs in this test don't match the description, but I think the test case itself is OK. It's nice to cover both matching and unmatching NodeIDs in the next two tests